requestsEx.py

```diff
@@ -0,0 +1,25 @@
+import requests
+import sys
+
+
+def top_repository_from_github(programming_language: str, number: int):
+    """
+
+    The function receives a language and a number, and brings from the GitHub the number of repositories according
+    to the number and language received, arranged according to the number of stars in descending order.
+    :param programming_language: The language in which you want the repositories.
+    :param number: The language in which you want the repositories.
+    """
+    headers = {'Accept': 'application/vnd.github.v3+json'}
+    username = sys.argv[1]
```

Comment:
Separate io from logic - You are receiving arguments from the command line (good job on that!) and printing the results to the console. What if you would have wanted to get the username and the token from somewhere else and export the results to a place other then the console? You wouldn't be able to do that because your IO is coupled to your logic. Read this link for more - 
https://wiki.c2.com/?SeparateIoFromCalculation

Reply:
Try separating this code to multiple functions